.github/workflows/generate.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         run: npm install -g @bufbuild/buf
 
       - name: Generate code
-        run: buf generate proto
+        run: make all
```

Comment:
I would prefer not to modify this step. We should put the manual replacement in a python-specific step.